src/lcgs/parse.rs

```diff
@@ -16,6 +18,27 @@ use crate::lcgs::ast::UnaryOpKind::{Negation, Not};
 use crate::lcgs::ast::*;
 use crate::lcgs::precedence::Associativity::RightToLeft;
 use crate::lcgs::precedence::{precedence, Precedence};
+use std::borrow::Borrow;
+use std::collections::HashSet;
+
+/// Required for static allocation of a hashset
+lazy_static! {
+    static ref RESERVED_KEYWORDS: HashSet<&'static str> = {
+        let mut set = HashSet::new();
+        set.insert("const");
+        set.insert("label");
+        set.insert("player");
+        set.insert("template");
+        set.insert("init");
+        set.insert("int");
+        set.insert("true");
+        set.insert("false");
+        set.insert("not");
+        set.insert("min");
+        set.insert("max");
+        set
```

Comment:
"not" and "int" are not keywords anymore. But "endtemplate" is.

Reply:
Fixed in 0176d54